src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Random random = new Random();
+        Lottery lottery = new Lottery(colorSupplier, random);
```

Comment:
If you send Random object in Lottery constructor, why you don`t do it for ColorSupplier?